core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -524,10 +525,10 @@ private static String toSql(RelNode root, SqlDialect dialect,
 
   @Test void testSelectQueryWithHiveRollup() {
     String query = "select \"product_class_id\", \"product_id\", count(*) "
-            + "from \"product\" group by rollup(\"product_class_id\", \"product_id\")";
+        + "from \"product\" group by rollup(\"product_class_id\", \"product_id\")";
```

Comment:
Please avoid unnecessary higher values in order to preserve the original Git history.

Reply:
done, removed unnecessary changes